src/esaml_sp.erl

```diff
@@ -315,6 +315,8 @@ decrypt_key_info(EncryptedData, Key) ->
     CipherValue = base64:decode(CipherValue64),
     decrypt(CipherValue, Algorithm, Key).
 
+decrypt(CipherValue, "http://www.w3.org/2001/04/xmlenc#rsa-1_5", Key) ->
+    public_key:decrypt_private(CipherValue, Key);
```

Comment:
Can you make the padding explicit with the 3rd parameter? I think it is `rsa_pkcs1_padding`.